readme.md

```diff
@@ -34,6 +34,14 @@ Due to security considerations, build logs are not publicly available.
 | Microsoft.Azure.Devices.DigitalTwin.Client            | N/A                                                       | [![NuGet][pnp-device-prerelease]][pnp-device-nuget]           |
 | Microsoft.Azure.Devices.DigitalTwin.Service           | N/A                                                       | [![NuGet][pnp-service-prerelease]][pnp-service-nuget]         |
 
+> Note:    
+The device streaming feature is no longer being shipped as part of our newer preview releases as there is no active development going on for this feature in the service. The feature has not been included in any preview release after [2020-10-14](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/preview_2020-10-14). However, the feature is still available under previews/deviceStreaming branch.  
```

Comment:
"Device streaming preview feature has not been included in our newer preview releases..."
We can also add a link to the doc: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-streams-overview

Might also be worth adding a note that prod application should not take dependency on preview nugets - these can include breaking changes

Reply:
All good points. Let me add them.